### ISSUE TITLE
False positive for date-fns

### ISSUE THREAD

**Issue by ilyub** [benmosher/eslint-plugin-import#2113]
```ts
import enGB from "date-fns/locale/en-GB";
import enUS from "date-fns/locale/en-US";
```

I get this warning:
'...\node_modules\date-fns\typings.d.ts' imported multiple times. eslint (import/no-duplicates)

**Comment by ljharb** [comment]
This is a duplicate of #2068.
